src/Skia/Avalonia.Skia/Gpu/OpenGl/GlSkiaExternalObjectsFeature.cs

```diff
@@ -240,4 +247,31 @@ public IBitmapImpl SnapshotWithAutomaticSync()
         using (_gpu.EnsureCurrent())
             return TakeSnapshot();
     }
+
+    private int CopyToNewTexture(int textureType, int sourceTextureId, int internalFormat, int width, int height)
+    {
+        var gl = _gpu.GlContext.GlInterface;
+
+        using var _ = _gpu.EnsureCurrent();
+
+        if (_fbo == 0)
+            _fbo = gl.GenFramebuffer();
+
+        // Bind source texture
+        gl.BindFramebuffer(GL_FRAMEBUFFER, _fbo);
```

Comment:
It _might_ be a good idea to save previous `GL_FRAMEBUFFER`(`_BINDING`) / `GL_READ_FRAMEBUFFER`(`_BINDING`) and `GL_SCISSOR_TEST` values and restore those afterwards.

Not necessary right now since the entire texture import operation currently happens before Skia calls, but it could save us a `GrContext.Reset()` call later